docs/references/nextjs/trpc.mdx

```diff
@@ -110,11 +123,11 @@ export const exampleRouter = router({
 })
 ```
 
-## Using tRPC Middleware
+## Update your tRPC Middleware
 
-In most cases, you need to protect your routes from users who aren't signed in. You can create a protected procedure.
+In most cases, you need to protect your routes from users who aren't signed in. You can create a protected procedure by updating your tRPC middleware like so:
```

Comment:
```suggestion

In many applications, it's essential to restrict access to certain routes based on user authentication status. This ensures that sensitive data and functionality are only accessible to authorized users. tRPC middleware provides a powerful mechanism for implementing this protection within your application.

### Creating a Protected Procedure:

Follow these steps to create a procedure that requires user authentication:
```
@alexisintech thoughts on the rewording?

Reply:
I'd change the heading to `Update your tRPC middleware` and the heading above this one to read `Create a Protected Procedure`. Because updating your tRPC middleware is a step within creating a protected procedure.

and then instead of the "Follow these steps" line, which is a little vague, I'd write:

To create a protected procedure, you first need to update your tRPC middleware. In the example below, the `isAuthed` helper uses the context (`ctx`) to get access to the user's ID. If the user's ID exists in the authentication context, this means that the user is signed in. If the user is not signed in, an `UNAUTHORIZED` error is thrown.